### ISSUE TITLE
Error when adding Slack Notifier w/ same webhook

### ISSUE THREAD

**Issue by Antonlovesdnb** [aol/moloch#1329]
ES Version: 6.8.2
Moloch Version: 2.1.0
OS: Ubuntu 18.04.3 LTS
Moloch Installed: deb

To reproduce: 

Add a Slack Notifier in Parliament --> Add a new Slack Notifier in Moloch with the same webhook URL, but different name.

via viewer.log:

> Trying to add duplicate notifier { userId: '_moloch_shared',
>   userName: '_moloch_shared',
>   enabled: false,
>   webEnabled: false,
>   emailSearch: false,
>   headerAuthEnabled: false,
>   createEnabled: false,
>   removeEnabled: false,
>   packetSearch: false,
>   views:
>    { IDS:
>       { expression:
>          'suricata.signature== EXISTS! && ip.dst <redacted>,
>         user: 'anton',
>         shared: true,
>         sessionsColConfig: [Object] } },
>   notifiers:
>    { Slack: { name: 'Slack', type: 'slack', fields: [Array] },
>      SlackMoloch: { name: 'SlackMoloch', type: 'slack', fields: [Array] },
>      Slack12: { name: 'Slack12', type: 'slack', fields: [Array] },
>      MolochSlack: { name: 'MolochSlack', type: 'slack', fields: [Array] } } }

**Comment by 31453** [comment]
I am unable to reproduce this issue given the steps provided. Parliament and Moloch store their notifiers in completely different places, so your steps are **likely** a red herring.

It looks like you're trying to add a notifier to Moloch that already exists with that name. Are you trying to add a notifier with the name "Slack", "SlackMoloch", "Slack12", or "MolochSlack"? Do you see those 4 notifiers in the Moloch UI? If not, that's the issue.

**Comment by Antonlovesdnb** [comment]
Yesterday when I checked the UI I only saw one notifier in parliament and none in Moloch. Today I restarted viewer and saw additional entries for notifiers in the UI in Moloch, when I deleted those I was able to add a notifier in Moloch with a different name than Parliament  - probably a caching issue on the browser on my end maybe? Unless anything in the above leads you to believe it's a bug on the Moloch end, from my end everything is working as expected now - thank you!
